src/styles/styles.js

```diff
@@ -783,6 +783,15 @@ const styles = {
         justifyContent: 'center',
     },
 
+    chatItemTopPadding: {
+        backgroundColor: 'rgba(0,0,0,0)',
+        textAlignVertical: 'top',
+        paddingTop: 0,
```

Comment:
Is there some default padding being applied here that we are removing? Maybe we should not apply it in the first place?

Reply:
`textInputCompose` on line 763 has `paddingTop: 10,` which caused bug #1002  on Android .

Reply:
Not sure I follow this explanation. Removing the `paddingTop` there doesn't seem to have any change.

Reply:
You are right on that. `paddingTop` should be removed. Sorry I didn't clearly get the question at first glance. Thanks for noticing .

Reply:
kindly test on `Android`

Reply:
I have tested this on Android. And this does not seem related to the padding values at all.